src/ListingPage.php

```diff
@@ -47,6 +47,9 @@ private function update_context(): void
         $this->context['page_category'] = is_home() ? 'News' : 'Listing Page';
         $this->context['og_type'] = isset($this->context['author']) ? 'profile' : 'website';
 
+        // Filters (News & Stories page only)
+        $this->set_filters();
```

Comment:
This is only a personal suggestion:

```suggestion
        $this->context['current_category'] = is_home() && isset($_GET['category']) && $_GET['category'] ? $_GET['category'] : '';
```